src/test/java/de/retest/web/it/EmptyDivTestIT.java

```diff
@@ -0,0 +1,46 @@
+package de.retest.web.it;
+
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.openqa.selenium.WebDriver;
+
+import de.retest.recheck.Recheck;
+import de.retest.recheck.RecheckImpl;
+
+class EmptyDivTestIT {
```

Comment:
Instead of yet another IT on top of Selenium, does a unit test make sense? As I see, only `WebData` has been changed? At least it looks like `isShown()` can be easily tested.